src/mem/metaslab.h

```diff
@@ -92,27 +92,28 @@ namespace snmalloc
     }
 
     /// Value used to check for corruptions in a block
-    static constexpr size_t POISON = 
+    static constexpr size_t POISON =
       static_cast<size_t>(bits::is64() ? 0xDEADBEEFDEAD0000 : 0xDEAD0000);
 
     /// Store next pointer in a block. In Debug using magic value to detect some
     /// simple corruptions.
     static void store_next(void* p, uint16_t head)
-    { 
+    {
 #ifdef NDEBUG
```

Comment:
It would be nice to make this a separate macro from `NDEBUG`, because some users may wish to use this in production and I may wish to debug the other code path at some point.